src/config/crd/bases/goharbor.goharbor.io_assessmentreports.yaml

```diff
@@ -143,8 +143,8 @@ spec:
                       governor:
                         description: Indicate whether to config of governor
                         properties:
-                          apiToken:
-                            description: Api token for user authentication
+                          cspSecretName:
```

Comment:
Change it to "Secret name where CSP api token is stored in cnsi-system namespace"